bookshop/src/store/index.js

```diff
@@ -40,23 +42,34 @@ const store = new Vuex.Store({
     async booklist ({commit}) {
       try {
         let {data: {data: res}} = await axios.get('getBooks')
-        commit('set', {type: 'books', items: res})
+        commit('set', {type: 'books', data: res})
```

Comment:
make it more specific. Let it be 'books'
let {data: {data: books}} = await axios.get('getBooks')
commit('set', {type: 'books', data: books})

Reply:
Fixed